src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -394,7 +394,7 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] testProvidersChain() {
+  public static Object[][] providersChainOasis() {
     return new Object[][]{
         {
             "adtest", "SyntheticTests/ProvidersChain",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/ProvidersChain" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)